examples/server-history.py

```diff
@@ -34,12 +42,17 @@
 
     # enable history for this particular node, must be called after start since it uses subscription
     server.iserver.enable_history(myvar, period=None, count=100)
+
+    # enable history for myobj events generated by myevgen
+    server.iserver.enable_event_history(myobj, myevgen.event, period=None)
```

Comment:
as written ealier I think we should pass a list which can be either a list of properties to store, a list of events or a list of event type